modules/sfm/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 set(OPENCV_MODULE_IS_PART_OF_WORLD FALSE)
 set(the_description "SFM algorithms")
 
+if(${CMAKE_VERSION} VERSION_LESS 3.0)
+  message(STATUS "Module opencv_sfm has been disabled: does not support cmake version less than 3.0")
+  ocv_module_disable(sfm)
```

Comment:
Actually it depends on external packages, so it can't be (and should not be) handled in OpenCV.

Ubuntu 14.04 with CMake 2.8.12.2 [works](http://pullrequest.opencv.org/buildbot/builders/master-contrib-lin64/builds/10350/steps/cmake/logs/stdio) well.

Reply:
I've reverted this change.